Formula/fisherman.rb

```diff
@@ -0,0 +1,18 @@
+class Fisherman < Formula
+  desc "fish plugin manager"
+  homepage "https://github.com/fisherman/fisherman"
+
+  url "https://raw.githubusercontent.com/fisherman/fisherman/2.12.0/fisher.fish"
```

Comment:
The tarball should be used (https://github.com/fisherman/fisherman/archive/2.13.0.tar.gz) and 2.13.0 is now the latest.